src/main/java/org/jabref/JabRefMain.java

```diff
@@ -31,11 +31,10 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-/**
- * JabRef MainClass
- */
+// JabRef MainClass
```

Comment:
Why do you think this is better than the Javadoc comment?

Reply:
Some of the large organisations use '//' to mark all the comments whether it be Doc comment or something else.
The main purpose is the reduction of three lines into one lines which proves beneficial for their large amount of code.
If your organisation prefers this pattern, I have no issue.